docs/generators/fsharp-functions.md

```diff
@@ -22,7 +22,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |prependFormOrBodyParameters|Add form or body parameters to the beginning of the parameter list.| |false|
 |sortModelPropertiesByRequiredFlag|Sort model properties to place required parameters before optional parameters.| |true|
 |sortParamsByRequiredFlag|Sort method arguments to place required parameters before optional parameters.| |true|
-|sourceFolder|source folder for generated code| |OpenAPI/src|
+|sourceFolder|source folder for generated code| |OpenAPI\src|
```

Comment:
Why were these non-python files modified?
What os are you on?

Reply:
Windows 10

Reply:
I will revert the changes happened during build to non python code

Reply:
@spacether revert the changes done to non python-flask generators

Reply:
@spacether after reverting the changes in non python generators checks are failing